pkg/comp-functions/functions/vshnforgejo/deploy.go

```diff
@@ -173,10 +173,10 @@ func addForgejo(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.V
 		"postgresql-ha": map[string]any{
 			"enabled": false,
 		},
-		"redis": map[string]any{
+		"valkey": map[string]any{
```

Comment:
what is this? valkey?

Reply:
In the new chart they are replacing redis with valkey which is a fork of redis. Have to disable the valkey things otherwise it will spin up unnecessary Pods